dask.Dockerfile

```diff
@@ -17,9 +17,9 @@ RUN conda install -c gpuci gpuci-tools
 
 RUN gpuci_conda_retry install -c conda-forge mamba
 
-RUN mamba env create -n dask --file /dask_environment.yaml
+RUN gpuci_mamba_retry env create -n dask --file /dask_environment.yaml
```

Comment:
Just for my own understanding, does `gpuci_mamba_retry` add some retry logic around normal `mamba` commands? For example, to handle random HTTP errors

Reply:
Yes, that's correct.